src/Microsoft.AspNetCore.Rewrite/ModRewriteExtensions.cs

```diff
@@ -13,76 +13,57 @@ public static class ModRewriteExtensions
         /// <summary>
         /// Imports rules from a mod_rewrite file and adds the rules to current rules. 
         /// </summary>
-        /// <param name="options">The UrlRewrite options.</param>
-        /// <param name="hostingEnv"></param>
+        /// <param name="options">The Rewrite options.</param>
+        /// <param name="hostingEnvironment">The Hosting Environment</param>
         /// <param name="filePath">The path to the file containing mod_rewrite rules.</param>
-        public static RewriteOptions ImportFromModRewrite(this RewriteOptions options, IHostingEnvironment hostingEnv, string filePath)
+        public static RewriteOptions ImportFromModRewrite(this RewriteOptions options, IHostingEnvironment hostingEnvironment, string filePath)
         {
             if (options == null)
             {
                 throw new ArgumentNullException(nameof(options));
             }
 
-            if (hostingEnv == null)
+            if (hostingEnvironment == null)
             {
-                throw new ArgumentNullException(nameof(hostingEnv));
+                throw new ArgumentNullException(nameof(hostingEnvironment));
             }
 
             if (string.IsNullOrEmpty(filePath))
             {
                 throw new ArgumentException(nameof(filePath));
             }
 
-            var path = Path.Combine(hostingEnv.ContentRootPath, filePath);
+            var path = Path.Combine(hostingEnvironment.ContentRootPath, filePath);
             using (var stream = File.OpenRead(path))
             {
-                options.Rules.AddRange(new FileParser().Parse(new StreamReader(stream)));
-            };
-            return options;
+                return ImportFromModRewrite(options, new StreamReader(stream));
```

Comment:
since it is an extension already, how about: `return options.ImportFromModeRewrite(new StreamReader(stream));`